lib/tasks/annotate_models.rake

```diff
@@ -47,6 +47,7 @@ task annotate_models: :environment do
   options[:ignore_routes] = ENV.fetch('ignore_routes', nil)
   options[:hide_limit_column_types] = Annotate.fallback(ENV['hide_limit_column_types'], '')
   options[:hide_default_column_types] = Annotate.fallback(ENV['hide_default_column_types'], '')
+  options[:ignore_unknown_models] = ENV["ignore_unknown_models"]
```

Comment:
In keeping with the style of all the others (assuming you're going to open a pull request upstream?), perhaps something like `ENV.fetch('ignore_unknown_models', false)`

Reply:
It looks to me like both ways are represented. I thought the option that doesn't throw the error may be favorable

Reply:
fetch doesn't throw an error, it provides an explicit default

Reply:
File a defect on the annotate repository and open a PR agains it; see what feedback they have as far as getting it merged upstream.